### PR TITLE
fix: correct building unit correct realization error message

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs [hunk 1]
```diff
@@ -69,7 +69,7 @@ public static string BuildingInvalid(string buildingPuri)
             public const string BuildingUnitCannotBeRealized = "Deze actie is enkel toegestaan op gebouweenheden met status 'gepland'.";
             public const string BuildingUnitCannotBeNotRealized = "Deze actie is enkel toegestaan op gebouweenheden met status 'gepland'.";
             public const string BuildingUnitCannotBeCorrectedFromNotRealizedToPlanned = "Deze actie is enkel toegestaan op gebouweenheden met status 'nietGerealiseerd'.";
-            public const string BuildingUnitCannotBeCorrectedFromNotRealizedToPlannedBecauseOfInvalidBuildingStatus = "Deze actie is enkel toegestaan voor een 'gepland' of 'gerealiseerd' gebouw.";
+            public const string BuildingUnitCannotBeCorrectedFromNotRealizedToPlannedBecauseOfInvalidBuildingStatus = "Deze actie is enkel toegestaan binnen een gepland of gerealiseerd gebouw.";
```
Comment (emalfroy) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs:72]: Doesn't there need to be ' between the statusses?

File: test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitRealization/GivenBuildingHasInvalidStatus.cs [hunk 2]
```diff
@@ -55,8 +55,8 @@ public void ThenValidationException()
                 .Result
                 .Where(x =>
                     x.Errors.Any(
-                        failure => failure.ErrorCode == "GebouwNietgerealiseerdOfGehistoreerd"
-                                    && failure.ErrorMessage == "Deze actie is enkel toegestaan voor een 'gepland' of 'gerealiseerd' gebouw."));
+                        failure => failure.ErrorCode == "GebouwStatusNietInGeplandOfGerealiseerd"
+                                   && failure.ErrorMessage == "Deze actie is enkel toegestaan binnen een gepland of gerealiseerd gebouw."));
```
Comment (emalfroy) [inline test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitRealization/GivenBuildingHasInvalidStatus.cs:59]: Doesn't there need to be ' between the statusses?

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs (hunk 1 above)
Comment (jvandaal) [reply src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs:72]: Nope. Already checked that with Anaïs.

File: test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitRealization/GivenBuildingHasInvalidStatus.cs (hunk 2 above)
Comment (jvandaal) [reply test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitRealization/GivenBuildingHasInvalidStatus.cs:59]: Nope. Already checked that with Anaïs.

**Pull request merged**